### PR TITLE
Set windows containers default network mode to 'nat'

### PR DESCRIPTION
**Pull request by mwieczorek**
As discussed in #1488: setting default network mode for windows containers to 'nat'

### CODE REVIEW

**Comment by diptanu** [comment]
LGTM

**Comment by diptanu** [comment]
@mwieczorek Can you also update the CHANGELOGs please?

File: client/driver/docker_default.go [hunk 1]
```diff
@@ -4,6 +4,10 @@ package driver
 
 import docker "github.com/fsouza/go-dockerclient"
 
+const (
+	defaultNetworkMode = "bridge"
```
Comment (dadgar) [inline client/driver/docker_default.go]: Put comments on these

File: client/driver/docker_windows.go [hunk 2]
```diff
@@ -2,6 +2,11 @@ package driver
 
 import docker "github.com/fsouza/go-dockerclient"
 
+const (
+	//Default network mode for windows containers is nat
```
Comment (dadgar) [inline client/driver/docker_windows.go]: In the future please put a space after the `//`

**Pull request merged**